packages/opentelemetry-id-generator-aws-xray/README.md

```diff
@@ -38,16 +38,15 @@ const tracerProvider = new NodeTracerProvider(tracerConfig);
 
 Example trace ID format: 58406520a006649127e371903a2de979
 
-A trace ID consists of two parts; the timestamp and the unique identifier.
+A trace ID consists of two parts: the timestamp and the unique identifier.
 
 #### Time Stamp
```

Comment:
I know changing the docs wasn't the intention of the PR, but it caught my eye since you did changes in the readme file as well. :laughing: 

```suggestion
#### Timestamp
```

Reply:
Since you were at it, I'd actually just format the whole section to be more concise:

---

A trace ID consists of two parts: 

1. **Timestamp**: The first 8 hexadecimal digits represent the time of the original request in Unix epoch time. For example, 10:00 AM December 1st, 2016 PST in epoch time is 1480615200 seconds, or 58406520 in hexadecimal digits.
2. **Unique Identifier**: The last 24 hexadecimal digits is an random identifier for the trace.

---

But that comes to style more than correctness.